packages/expo-cli/src/commands/prebuild.ts

```diff
@@ -0,0 +1,47 @@
+import chalk from 'chalk';
+import { Command } from 'commander';
+
+import * as Eject from './eject/Eject';
+import { platformsFromPlatform } from './eject/platformOptions';
+import { learnMore } from './utils/TerminalLink';
+
+export async function actionAsync(
+  projectDir: string,
+  {
+    platform,
+    ...options
+  }: Eject.EjectAsyncOptions & {
+    npm?: boolean;
+    platform?: string;
+  }
+) {
+  if (options.npm) {
+    options.packageManager = 'npm';
+  }
+
+  // Clear the native folders before syncing
+  await Eject.clearNativeFolder(projectDir, ['ios', 'android']);
+
+  await Eject.prebuildAsync(projectDir, {
+    ...options,
+    platforms: platformsFromPlatform(platform),
+  } as Eject.EjectAsyncOptions);
+}
+
+export default function (program: Command) {
+  program
+    .command('prebuild [path]')
+    .description(
+      `Create native iOS and Android project files before building natively. ${chalk.dim(
```

Comment:
let's mark this as experimental so it's clear that people shouldn't depend on it yet